### ISSUE TITLE
Have you solved the problem?

### ISSUE THREAD

**Issue by ruogudu** [algobardo/sootReproduce#1]
Hi! 
I think I've encountered the same problem when instrumenting android frameworks with soot. I tried on services and the system won't boot after I switched the services.jar. I've run out of my wits. If you have solved the problem, please tell me. Thank you!

**Comment by algobardo** [comment]
If I remember correctly one of the biggest problem was the fact that the dex format limits the number of methods in a dex file and soot was not supporting the splitting. 
This looks like to be an issue still open:
https://github.com/Sable/soot/issues/565

I also refer you to the soot mailing list, and to my original thread (https://mailman.cs.mcgill.ca/pipermail/soot-list/2014-September/007278.html).

Feel also free to drop me an email (http://gmezzetti.name contains contact info)

**Comment by ruogudu** [comment]
Thanks for sharing! I just found out I was facing a simpler problem than you. It was because soot would automatically generate some placeholder methods when their bodies are missing, probably in their super class. What do these placeholder methods do is just to throw a exception. That's probably why the zygote was killed. So I removed them all and problem solved. Anyway, Thank you again for your help and I hope this issue can help others too.
